src/main/java/com/rakovets/course/java/core/practice/looping_statements/Task02.java

```diff
@@ -34,6 +40,10 @@ public static void main(String[] args) {
     static double getTotalDepositAmount(double depositAmount, double annualDepositPercent, int depositTerm) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0;
+        for (int i = 1; i <= depositTerm; i++) {
+            double profit = depositAmount * (annualDepositPercent / 100);
+            depositAmount  += profit;
+        }
+            return NumberUtil.roundValueToTwoDigitsForMantissa(depositAmount);
```

Comment:
"Violation: There aren't correct indentations for inner code
Solution: Add correct indentations for inner code"